server/validation/prospect.js

```diff
@@ -0,0 +1,39 @@
+const Validator = require("validator");
+const isEmpty = require("is-empty");
+
+const { Status } = require("../models/Prospect")
+
+module.exports = function validateProspectInput(data) {
+  let errors = {};
+
+// Convert empty fields to an empty string so we can use validator functions
+  data.first_name = !isEmpty(data.first_name) ? data.first_name : "";
+  data.last_name = !isEmpty(data.last_name) ? data.last_name : "";
+  data.email = !isEmpty(data.email) ? data.email : "";
+  data.status = !isEmpty(data.status) ? data.status : "";
+
+// Name checks
+  if (Validator.isEmpty(data.first_name)) {
```

Comment:
Do we need these checks if above you are setting them to "" if they are not empty?